test/components/Provider.spec.js

```diff
@@ -75,47 +81,52 @@ describe('React', () => {
 
     it('should add the store to the child context', () => {
       const store = createStore(() => ({}))
+      store.mine = 'hi'
```

Comment:
I'm not sure this is the best approach for identifying a store. It would be better to provide canary values in the initialState for each store. Setting arbitrary properties is technically possible, but it's not idiomatic Redux and might be confusing to those working with these tests in the future.

Reply:
I'm fixing this in the 16.x PR I'm making. Fully agree, and an easy fix

Reply:
I already took care of this in the #998 branch - I put some canary values in the store as suggested.